appserver/website-bootstrap/index.html

```diff
@@ -94,13 +99,13 @@ <h3 class="side-title">Top News</h3>
 	    </div>
         <footer class="page-footer footer container-fluid">
             <ul class="team-members">
-                <li><a href="">Jennifer Chao</a></li>
-                <li><a href="">Conor Kennedy</a></li>
-                <li><a href="https://www.youtube.com/watch?v=dQw4w9WgXcQ">Mike Lauw</a></li>
-                <li><a href="https://www.youtube.com/watch?v=ndcDT1rXMvU&list=PLKtIunYVkv_SUyXj_6Fe53okfzM9yVq1F">Chris Nelson</a></li>
-                <li><a href="">Scott Salisbury</a></li>
-                <li><a href="">Dan Tran</a></li>
-                <li><a href="">Thanh Tran</a></li>
+                <li>Jennifer Chao</li>
+                <li>Conor Kennedy</li>
+                <li>Mike Lauw</li>
+                <li>Chris Nelson</li>
+                <li>Scott Salisbury</li>
+                <li>Dan Tran</li>
+                <li>Thanh Tran</li>
```

Comment:
why no links?

Reply:
no one seemed interested, we can add them easily later if people are interested